numba/cuda/codegen.py

```diff
@@ -24,7 +24,7 @@ def disassemble_cubin(cubin):
             f.write(cubin)
 
         try:
-            cp = subprocess.run(['nvdisasm', fname], check=True,
+            cp = subprocess.run(['nvdisasm', '-gi', fname], check=True,
```

Comment:
Could you just add a comment mentioning that `-gi` adds line info? Or maybe better yet, do `lineinfo_flag = '-gi'` on the previous line and use that in the list of arguments, so the code is self-explaining?